src/components/CharacterTab.jsx

```diff
@@ -222,6 +232,33 @@ export default function CharacterTab() {
     cellStyle: { display: 'flex' },
   }), [])
 
+  const externalFilterChanged = useCallback((newValue) => {
+    setCurrentFilters(newValue)
+    gridRef.current.api.onFilterChanged()
+  }, [])
+
+  const charinfo = JSON.parse(JSON.stringify(DB.getMetadata().characters))
```

Comment:
This probably shouldn't be outside the callback, we don't need this on every render. Should be a Utils.clone